context_test.go

```diff
@@ -0,0 +1,34 @@
+package gb
+
+import (
+	"runtime/debug"
+	"strings"
+	"testing"
+)
+
+func testImportCycle(pkg string, t *testing.T) {
```

Comment:
Ok, but will leave stack limit there. Just in case if we break it in the future so we don't lose our heads on finding the issue.
Sounds good?